x-pack/plugins/apm/public/components/shared/Stacktrace/index.tsx

```diff
@@ -79,16 +79,17 @@ export class Stacktrace extends PureComponent<Props, State> {
                 />
               );
             }
-            return groupedStackframes.map((stackframe, idx) =>
-              hasSourceLines(stackframe) ? (
-                <CodePreview
-                  key={idx}
-                  stackframe={stackframe}
-                  codeLanguage={codeLanguage}
-                />
-              ) : (
-                <FrameHeading key={idx} stackframe={stackframe} />
-              )
+            return groupedStackframes.map(
+              (stackframe, idx) =>
+                hasSourceLines(stackframe) ? (
+                  <CodePreview
+                    key={idx}
+                    stackframe={stackframe}
+                    codeLanguage={codeLanguage}
+                  />
+                ) : (
+                  <FrameHeading key={idx} stackframe={stackframe} />
+                )
```

Comment:
Did we upgrade prettier or why is this formatting change happening?

Reply:
mea culpa @sqren 

ran a global lint, and this shouldn't have snuck into this PR. apologies for the noise.

Reply:
No worries :)